src/BuildScriptGenerator/Php/PhpBashBuildSnippet.sh.tpl

```diff
@@ -0,0 +1,13 @@
+﻿echo "PHP executable: $php"
+
+cd "$SOURCE_DIR"
+
+{{ if ComposerFileExists }}
+declare -r composer='/opt/php-composer/composer.phar'
+echo "Composer archive: $composer"
+echo "Running composer install..."
+echo
+$php $composer install
```

Comment:
q: why do we need to use `$php` instead of just `php`? In pre/post build scripts, the user should be able to use just `php` and it must be in the right version